tests/integration/minion/test_executor.py

```diff
@@ -12,6 +13,7 @@ def setup(self):
         self.run_function("saltutil.sync_all")
 
     @pytest.mark.slow_test
+    @pytest.mark.skipif(sys.version_info.minor==11, reason="Flaky with Python 3.11")
```

Comment:
Hmmm, what about using:

```suggestion
    @pytest.mark.flaky(max_runs=4)
```

Reply:
Unfortunatelly it doesn't work, I tried it with 10 and it keeps failing, but as I mentioned only in case of running inside Salt Shaker environment.